src/parser/jobs/blu/modules/ColdFog.tsx

```diff
@@ -0,0 +1,146 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {History} from 'parser/core/modules/ActionWindow/History'
+import {Actors} from 'parser/core/modules/Actors'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+// Tiered suggestion severities
+const COLD_SEVERITY = {
+	DROPPED_COLD_FOG: {
+		1: SEVERITY.MAJOR,
+	},
+	DROPPED_WHITE_DEATH_CASTS: {
+		1: SEVERITY.MEDIUM,
+	},
+	INEFFECTIVE_COLD_FOG: {
+		1: SEVERITY.MAJOR,
+	},
+}
+
+const COLD_FOG_MINIMUM_VIABLE_CASTS = 2
+const COLD_COLD_IDEAL_CASTS = 6
+
+interface ColdFogWindow {
+	whiteDeathCasts: number
+}
+
+export class ColdFog extends Analyser {
+	static override handle = 'coldfog'
+	static override title = t('blu.cold_fog.title')`Cold Fog`
+
+	@dependency private actors!: Actors
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private touchOfFrostProcs = 0
```

Comment:
Looks like this variable isn't used anymore either?

Reply:
Patched in 4c257c4f7d9a3c05b398253c8f79da1d9557397d